projects/wget2/Dockerfile

```diff
@@ -17,27 +17,40 @@
 FROM gcr.io/oss-fuzz-base/base-builder
 MAINTAINER rockdaboot@gmail.com
 RUN apt-get update && apt-get install -y \
+ make \
  pkg-config \
  gettext \
+ autogen \
  autopoint \
  autoconf \
  automake \
  libtool \
  texinfo \
  flex \
- liblzma-dev \
- libidn11-dev \
- libidn2-0-dev \
- libunistring-dev \
- zlib1g-dev \
- libbz2-dev \
- gnutls-dev \
- libpsl-dev \
- libnghttp2-dev \
- doxygen
+ bison \
+ doxygen \
+ gtk-doc-tools \
+ gettext \
+ gengetopt \
+ curl \
```

Comment:
Are all of these needed? That's fine, I just want to double check that it is not a leftover from local debugging :)